lib/compress.js

```diff
@@ -148,10 +148,19 @@ merge(Compressor.prototype, {
             node.process_expression(true);
         }
         var passes = +this.options.passes || 1;
-        for (var pass = 0; pass < passes && pass < 3; ++pass) {
+        var last_count;
```

Comment:
The implicit `undefined` is more stable than explicit `Infinity` (think `var Infinity = 42;`)

How about we meet in the middle and settle for `var last_count = 1/0;`? :wink: